chore(Upgrade guide): removed verbiage on button aria-disabled (#4494)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/get-started/upgrade.md

#### Button

1. *Cannot find aria-disabled*

>     - **Cause:** We changed button's `isDisabled` prop to assign a value for `disabled`, but none for `aria-disabled` - except for when the `component` prop is passed anything other than "button" as a value. Additionally, `aria-disabled` will now only render when true.

Reviewer: ```suggestion
    - **Cause:** We changed button's `isDisabled` prop to assign a value for `disabled`, but none for `aria-disabled`. The exception to this is when the `component` prop is passed anything other than "button" as a value. Additionally, `aria-disabled` will now only render when true.
```

>     - **Fix:** Remove tests that look for `aria-disabled` in buttons when the expectation is for it to be false, or to match `isDisabled` when `component="button"`.

Reviewer: ```suggestion
    - **Fix:** Remove tests that look for `aria-disabled` in buttons when the expectation is either: 
        - For `aria-disabled` to be false.
        - For `aria-disabled` to match `isDisabled` when `component="button"`.
```
I'm not sure if I'm correctly interpreting the end of the sentence. I took it like there are 2 instances where you should remove these tests, but I can also interpret it like this (I'll just share options for both): 

> **Fix:** Either:
>     - Remove tests that look for `aria-disabled` in buttons, when the expectation is for `aria-disabled` to be false.
>     - Match `aria-disabled` to `isDisabled` when `component="button"`.
